backend/modules/model_gateway/model_gateway.py

```diff
@@ -1,85 +1,119 @@
 import json
-from pydantic import BaseModel
-from typing import List, Optional
-from backend.types import ModelConfig, ModelProviderConfig, ModelType
+import os
+from typing import List
+
 from langchain.embeddings.base import Embeddings
-from langchain_openai import OpenAIEmbeddings
 from langchain_core.language_models.chat_models import BaseChatModel
+from langchain_openai import OpenAIEmbeddings
 from langchain_openai.chat_models import ChatOpenAI
-import os
+
+from backend.logger import logger
+from backend.types import ModelConfig, ModelProviderConfig, ModelType
+
 
 class ModelGateway:
     config: List[ModelProviderConfig]
     modelsToProviderMap = {}
-    def __init__(self):        
+
+    def __init__(self):
         with open("./models_config.json") as f:
             data = json.load(f)
             self.config = [ModelProviderConfig(**item) for item in data]
```

Comment:
nit: this should have been `.parse_obj(item)`

Reply:
done.